ACE/ace/Basic_Types.h

```diff
@@ -562,18 +562,21 @@ ACE_END_VERSIONED_NAMESPACE_DECL
 # endif /* ACE_SIZEOF_LONG_DOUBLE */
 
 // Max and min sizes for the ACE integer types.
-#define ACE_CHAR_MAX 0x7F
-#define ACE_CHAR_MIN -(ACE_CHAR_MAX)-1
-#define ACE_OCTET_MAX 0xFF
+#define ACE_INT8_MAX 0x7F
```

Comment:
We should convert these to `constexpr` at some point

Reply:
We should, but I'd like that to be later because I don't want to make this harder to backport than it already is.